preprocessing/nextclade/tests/test_processing_functions.py

```diff
@@ -661,5 +667,49 @@ def test_parse_date_into_range() -> None:
     ), "dateRangeLower: empty date should be returned as None."
 
 
+@pytest.mark.parametrize(
+    "case_dir",
+    ["cchf_1", "west_nile_1"],
+    ids=["cchf_1", "west_nile_1"],
+)
+def test_create_flatfile(config: Config, case_dir):
+    test_data_dir = os.path.join(os.path.dirname(__file__), "test_data", case_dir)
+
+    metadata_tsv_path = os.path.join(test_data_dir, "metadata.tsv")
+    metadata = {}
+    with open(metadata_tsv_path, encoding="utf-8") as f:
+        header = f.readline().strip().split("\t")
+        values = f.readline().strip().split("\t")
+        metadata = dict(zip(header, values, strict=False))
```

Comment:
we can use pandas for this :-) 
```suggestion
metadata = (
    pd.read_csv(Path(test_data_dir) / "metadata.tsv", sep="\t", nrows=1)
      .iloc[0]
      .to_dict()
)
```

Reply:
Ah, neat! much better, thanks.